frontend/src/modules/shared/actions.js

```diff
@@ -124,6 +103,8 @@ export const fetchListedData = () => async (dispatch, getState) => {
         Projects.listTable,
         ProcessMeasurements.listTable,
         Protocols.list,
+        Datasets.listTable,
+        DatasetFiles.listTable,
     ].map(a => dispatch(a())))
 
     // Lower priority - Fetch summaries
```

Comment:
fetchListedData should not be loading summaries - that should only be done in fetchSummariesData. You should remove this and then check everywhere that `fetchListedData` is used to add a call to fetchSummariesData (ie. TemplateFlow.js and App.js)